README.md

```diff
@@ -2,6 +2,8 @@
 
 Target user:
 Our target group is Ward Nurses. This product is for users who prefer typing quickly while having quick access to critical patient information fast.
-Value proposition:
 
+Value proposition:
 Nurses already familiar with typing at their workstations. With this product, nurses are able to quickly access critical patient information (emergency contacts, drug allergies etc). They could also log their activities, for each patient whenever they administer care.
+
+_This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org)._
```

Comment:
I think this could be placed in index.md instead under the acknowledgements section.

Reply:
LGTM, I think we can acknowledge on both ends.